analyze_test.go

```diff
@@ -377,6 +377,131 @@ func TestAnalyzeFlagArgs(t *testing.T) {
 	})
 }
 
+//nolint:funlen
+func TestAnalyze_zone_sync(t *testing.T) {
+	t.Parallel()
+	testcases := map[string]struct {
+		stmt    *Directive
+		ctx     blockCtx
+		wantErr bool
+	}{
+		"zone_sync in stream server context ok": {
+			&Directive{
+				Directive: "zone_sync",
```

Comment:
I assume these are just missing unit tests for other directives unrelated to the `zone_sync_ssl_conf_command`.

Reply:
Yes, I didn't add tests to exhaustively check all of the `zone_sync...` directives, but I spot checked a few alongside the one I'm adding here.